run/types/DeviceWrapper.ts

```diff
@@ -363,7 +360,7 @@ export class DeviceWrapper {
         attempt++;
         if (attempt >= maxRetries) {
           throw new Error(
-            `Longpress on message: ${textToLookFor} unsuccessful after ${maxRetries} attempts, ${error}`
+            `Longpress on message: ${textToLookFor} unsuccessful after ${maxRetries} attempts, ${error as string}`
```

Comment:
error.toString() instead? error shouldn't be a string, so casting it to string is not correct

Reply:
Still getting the same error on that, what should the error be instead of string?

Reply:
Updated :)

Reply:
![Screenshot 2024-10-18 at 4 25 43 pm](https://github.com/user-attachments/assets/4ecbfed2-d05c-4b2f-887b-c4cb386f8cdf)

Reply:
Have updated but github hasn't refreshed the code snippet it seems (and i don't have any uncommited code...)